webrtc.html

```diff
@@ -7739,9 +7776,23 @@ <h2>Methods</h2>
                   <a data-cite="!GETUSERMEDIA#track-ended">ended</a>.</p>
                 </li>
                 <li>
-                  <p>Set <var>transceiver</var>'s <a>[[\Stopped]]</a>
+                  <p>Set <var>transceiver</var>'s <a>[[\Stopping]]</a>
                   slot to <code>true</code>.</p>
```

Comment:
Should the "stop sending and receiving" algorithm also set the [[Direction]] to inactive?
- Generally speaking, if we are calling stop() then we have to initiate a negotiation, so it doesn't hurt to update [[Direction]].
- More specifically, if we are calling stop() in have-remote-offer, it makes more sense to answer with a=inactive than a=whatever considering we are not sending or receiving with the accepted m= section.

Reply:
I think I agree here. Will update PR.